cmd/rook/userfacing/multus/validation/config.go

```diff
@@ -0,0 +1,34 @@
+package validation
+
+import (
+	"fmt"
+
+	"github.com/rook/rook/pkg/daemon/multus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	configCmd = &cobra.Command{
+		Use:   "config",
+		Short: "Generate a validation test config file for different default scenarios.",
+		Args:  cobra.NoArgs,
+	}
+
+	simpleConfigCmd = &cobra.Command{
+		Use:   "simple",
+		Short: "Generate a simple default config file.",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			y, err := multus.NewDefaultValidationTestConfig().ToYAML()
+			if err != nil {
+				return err
+			}
+			fmt.Print(y)
```

Comment:
```suggestion
			fmt.Println(y)
```
I prefer fmt.Println, not sure if this make sense here.

Reply:
The string already has a newline, so another is not needed.